nodeSDK.njsproj

```diff
@@ -16,6 +16,13 @@
     <EnableTypeScript>true</EnableTypeScript>
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">11.0</VisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+    <StartWebBrowser>False</StartWebBrowser>
+    <Environment>AZURE_SUBSCRIPTION_ID=2cac2a14-ba6b-46a6-bce8-2d9781a41ba2
+CLIENT_ID=0e44d72c-4a92-4fd0-a640-d165dde5da87
+DOMAIN=microsoft.com
+APPLICATION_SECRET=KsQw6kI7QKLair8uZkqFkMblh2ryIkJB98usb6xr2wU=
```

Comment:
I don't think you need any edits to the ".proj" file to make tests pass. I assume these changes are for your local debugging. If that's the case remove this file from the changes.